tests/conftest.py

```diff
@@ -9,24 +14,51 @@ def autouse_django_db(db):
     pass
 
 
-@pytest.fixture
-def notification_template(settings):
+@pytest.fixture(autouse=True)
+def force_settings(settings):
     settings.LANGUAGES = (("fi", "Finnish"), ("en", "English"))
-    notifications.register("event_created", "Event created")
+    settings.LANGUAGE_CODE = "en"
+
+
+@pytest.fixture
+def notification_template():
+    notification_type = "event_created"
+    notifications.register(notification_type, "Event created")
     notifications.register("event_approved", "Event approved")
```

Comment:
Not related to this PR in any way, but I wonder if this is actually used for something, or could it be just removed 🙂

Reply:
Hmm yeah, I think it's not being used for anything, so it's safe to delete. 👍

Reply:
yes, probably can be deleted

Reply:
I will delete it after I merge the PR 👍